wallets/electrum/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id 'bisq.java-library'
+    id 'bisq.protobuf'
+}
+
+ext {
+    swaggerVersion = '2.2.0'
+    jerseyVersion = '3.0.4'
+}
+
+dependencies {
+    implementation("bisq:persistence")
+    implementation project(':core')
+    implementation libs.jackson.databind
+
+    implementation("org.glassfish.jersey.containers:jersey-container-jdk-http:$jerseyVersion")
```

Comment:
Is a servlet container needed or is only the http server needed? We use for the Rest API the jdk http server to limit dependencies. Maybe we can synchronize with REST API dependencies. @salokiam will have better overview in that area...

Reply:
Ah ok.